### PR TITLE
add a basic Inventory system

### PR DESCRIPTION
**Pull request by PurpleProg**
just the basic class for the inventory system
the classes Chest and Quest are just example, used for typing and testing for now. They must be rewrite later on.

### CODE REVIEW

**Comment by ultimateownsz** [comment]
src/GUI/inventory.py

I see you made this, but before this can be made. We have to adjust how the linter works.

It now checks the directories: py_version, pygame_version and tests. So we have to rename pygame to just GUI. Change py_version to CLI and leave tests just tests.

Also not forget to do this in the file names too when we import certain objects or classes.

**Comment by PurpleProg** [comment]
we can use git ls-files to make the liner works regardless of file names. I'll change the dir names and the worklfow.

**Comment by ultimateownsz** [comment]
> we can use git ls-files to make the liner works regardless of file names. I'll change the dir names and the worklfow.

Where do we put this in the files? Or how do we use this.

**Comment by PurpleProg** [comment]
i've  just included git ls-files to the workflow configuration

File: .github/workflows/lint_and_test.yml
Comment (ultimateownsz) [inline .github/workflows/lint_and_test.yml:1]: This looks like a better way to check the filenames.

We have to figure out what we can improve upon the linter. We can always improve the "workflow" or flow overall.

File: src/GUI/inventory.py
Comment (ultimateownsz) [inline src/GUI/inventory.py:1]: The basic inventory system looks good to me.

We have to test out what works better for the player, if we inherit the inventory class to the player or the other way around.

File: tox.ini
Comment (ultimateownsz) [inline tox.ini:1]: Thanks for fixing this.

File: main.py
Comment (ultimateownsz) [inline main.py:1]: This change makes the overall code more readable to the others who use the code for future PR's.

**Review by ultimateownsz — APPROVED** [review]

**Pull request merged**